sale_stock_medical_prescription_us/models/medical_prescription_order_line.py

```diff
@@ -44,53 +43,64 @@ class MedicalPrescriptionOrderLine(models.Model):
     @api.multi
     def _compute_dispense_remain(self):
         day_uom_id = self.env.ref('medical_medication.product_uom_day')
-        for rec_id in self:
-            if not rec_id.duration_uom_id or rec_id.duration <= 0:
+        for record in self:
+
+            if not record.duration_uom_id or record.duration <= 0:
                 continue
 
-            procurement_ids = \
-                rec_id.dispensed_ids.filtered(
+            procurements = \
+                record.dispensed_ids.filtered(
                     lambda r: r.state == 'done'
                 ).sorted(
                     key=lambda r: r.date_planned
                 )
 
-            if not procurement_ids:
-                rec_id.dispense_remain_qty, rec_id.dispense_remain_day = 0, 0
+            if not procurements:
+                record.dispense_remain_qty, record.dispense_remain_day = 0, 0
+                continue
+
+            if not record.qty:
                 continue
```

Comment:
@lasley I added this block to fix the other 0 division error which was happening [here](https://github.com/LasLabs/vertical-medical/blob/410f815bc43b7017556ce6ec732404b98aabf1b0/sale_stock_medical_prescription_us/models/medical_prescription_order_line.py#L103).